vagrant-kubernetes-istio/Troubleshooting.md

```diff
@@ -19,22 +19,22 @@ This file helps note down issues we have seen and how to debug them
    ```
    
 1. setup_vm.sh chooses port 5000 to push docker images to VM, and 8080 to forward kubectl requests to VM. If any of these ports is already used by other applications, you can change the port.
-  * Open Vagrantfile, find these two lines and change host port to the one you prefer. E.g. change `host: 5000` to `host: 5001`
+   * Open Vagrantfile, find these two lines and change host port to the one you prefer. E.g. change `host: 5000` to `host: 5001`
   
    ```bash
    config.vm.network "forwarded_port", guest: 5000, host: 5000, host_ip: "10.10.0.2"
    config.vm.network "forwarded_port", guest: 8080, host: 8080
    ```
-  After this change you can run `sh setup_vm.sh`.
+   After this change you can run `sh setup_vm.sh`.
 
-  * If you want to change port 5000 for pushing docker images to VM, then you also need to modify setup_dockerdaemon_linux.sh. Find the line below and change 10.10.0.2:5000 to 10.10.0.2:5001
+   * If you want to change port 5000 for pushing docker images to VM, then you also need to modify setup_dockerdaemon_linux.sh. Find the line below and change 10.10.0.2:5000 to 10.10.0.2:5001
    ```bash
    sudo sed -i 's/ExecStart=\/usr\/bin\/dockerd -H fd:\/\//ExecStart=\/usr\/bin\/dockerd -H fd:\/\/ --insecure-registry 10.10.0.2:5000/' /lib/systemd/system/docker.service
```

Comment:
sed command does not look correct? 
Only thing probably needed is to show to find this line:
--insecure-registry 10.10.0.2:5000 and change it to --insecure-registry 10.10.0.2:5001

Reply:
I think if the user changes this line in /lib/systemd/system/docker.service manually, then the user needs to run sudo systemctl daemon-reload and sudo systemctl restart docker as well. And these steps should take after running script setup_dockerdaemon_linux.sh.